stdlib/2/inspect.pyi

```diff
@@ -1,7 +1,27 @@
-from types import TracebackType, FrameType, ModuleType
-from typing import Any, Dict, Callable, List, Optional, Tuple, Union, NamedTuple, Type
+from types import CodeType, TracebackType, FrameType, ModuleType
+from typing import Any, Dict, Callable, List, NamedTuple, Optional, Sequence, Tuple, Type, Union
 
 # Types and members
+class EndOfBlock(Exception): ...
+
+class BlockFinder:
+    indent: int
+    islambda: bool
+    started: bool
+    passline: bool
+    last: int
+    def tokeneater(self, type: int, token: str, srow_scol: Tuple[int, int],
```

Comment:
Why don't you include `indecorator`, `decoratorhasargs`, `last`?

Reply:
I missed `indecorator` and `decoratorhasargs` because they're Python 3-specific - added now.  `last` is already present.